test/karma.debug.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en" id="html">
+<head>
+%X_UA_COMPATIBLE%
+  <title>DEBUG</title>
+  <meta charset="utf-8">
+  <!-- Karma serves this page from /context.html. Other files are served from /base -->
+  <link rel="stylesheet" href="/base/external/qunit/qunit.css" />
+  <link rel="stylesheet" href="/base/test/data/testsuite.css" />
+</head>
+<body id="body">
+  <div id="qunit"></div>
+
+  <!-- Start: jQuery Test HTML -->
+  <!-- this iframe is outside the #qunit-fixture so it won't waste time by constantly reloading; the tests are "safe" and clean up after themselves -->
+  <iframe id="loadediframe" name="loadediframe" style="display:none;" src="/base/test/data/iframe.html"></iframe>
+  <div id="qunit-fixture"></div>
+  <!-- End: jQuery Test HTML -->
+
+  <!-- Start: Karma boilerplate -->
+  <script src="/context.js"></script>
+  <script src="/debug.js"></script>
+  <script>
+    %CLIENT_CONFIG%
```

Comment:
Update: though we could use a kludgy test like `/^[/]debug\b/.test( location.pathname )`.